lib/ReactViews/Map/Panels/HelpPanel/HelpPanel.jsx

```diff
@@ -142,11 +142,11 @@ class HelpPanel extends React.Component {
                 "One of the key features of the NSW Spatial Digital Twin is the ability to add and view 3D data, including 3D imagery, or photo reality mesh. This is an interactive way to explore high resolution, 3D aerial imagery of some areas of NSW.",
```

Comment:
>One of the key features of the NSW Spatial Digital Twin

@reginapramesti can we make this text not specific to the NSW Digital Twin? I'm wondering if we could pull this text from the lang file or the config settings for the map name or something...

Reply:
In fact we need to think about how we handle all of those help items, they all reference NSW Digital Twin...

Reply:
That's true, I've been thinking about this for a while but kept forgetting because I didn't log an issue for it 😆 I'll log an issue and work on it, though probably will be a separate PR